fbpic/openpmd_diag/boosted_particle_diag.py

```diff
@@ -748,19 +744,29 @@ def interpolate_particles_to_lab_frame( self, slice_array, current_z_boost, t ):
         z_lab = self.gamma_boost*( z + self.beta_boost*c*t_cross )
 
         # Back-transformation of momentum
-        gamma = np.sqrt(1. + (ux**2 + uy**2 + uz**2))
+        gamma = 1./inv_gamma
```

Comment:
The above change is not crucial here (both operations are equivalent), but I guess the new one is slightly faster.